02-household-queries/chainlit-household-bot.py

```diff
@@ -328,7 +335,9 @@ async def on_click_upload_default_files(action: cl.Action):
     msg = cl.Message(content="Processing files...", disable_feedback=True)
     await msg.send()
 
-    ingest_call(vectordb)
+    settings = cl.user_session.get("settings")
+    embeddings = settings["embedding"]
+    ingest_call(vectordb, embeddings)
```

Comment:
Use consistent pluralization
```suggestion
    embedding = settings["embedding"]
    ingest_call(vectordb, embedding)
```